sdk/remoterendering/azure-mixedreality-remoterendering/pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-mixedreality-authentication</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-mixedreality-authentication;current} -->
+      <version>1.1.1</version> <!-- {x-version-update;com.azure:azure-mixedreality-authentication;current} -->
```

Comment:
The version tag has to be updated to `dependency` instead of `current` to get the builds to pass.

Reply:
That makes sense to me. I've fixed it.